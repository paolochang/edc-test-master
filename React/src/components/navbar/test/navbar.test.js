import React from "react";
import { render } from "@testing-library/react";
import Navbar from "../navbar";

describe("Navbar", () => {
  it("compares snapshot", () => {
    const component = render(<Navbar totalCount={4} />);
    expect(component.container).toMatchSnapshot();
  });
});
