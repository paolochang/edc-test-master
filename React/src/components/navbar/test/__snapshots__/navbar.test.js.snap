// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar compares snapshot 1`] = `
<div>
  <nav
    class="navbar"
  >
    <i
      class="navbar-logo fas fa-leaf"
    />
    <span>
      Habit Tracker
    </span>
    <span
      class="navbar-count"
    >
      4
    </span>
  </nav>
</div>
`;
